### ISSUE TITLE
Relay docs

### ISSUE THREAD

**Issue by sciencewhiz** [wpilibsuite/frc-docs#492]
There isn't any software documentation for relays.

**Comment by Starlight220** [comment]
How many teams still use relays?

**Comment by AustinShalit** [comment]
In 2019, 187 teams instantiated 270 relays.

They are a supported feature and should be added to the documentation

**Comment by Starlight220** [comment]
I agree with the fact that they need documentation, and more docs could increase the amount of users. 
However, what are FRC uses of relays? What relay-based FRC products exist?

**Comment by AustinShalit** [comment]
A handful of teams use relays to power their compressor rather than the PCM so they can use an auto resetting breaker. Relays are also an easy way to turn an LED light ring on and off.

**Comment by ItayZiv** [comment]
Relays big use are in various custom circuits

**Comment by sciencewhiz** [comment]
Relays can also be used for low current motors.
